addon/components/polaris-unstyled-link.js

```diff
@@ -92,25 +80,40 @@ export default Component.extend({
    * @default noop
    * @public
    */
-  onClick() {},
+  onClick() {}
 
-  /**
-   * @private
-   */
-  dataPolarisUnstyled: 'true',
+  /** @private */
+  dataPolarisUnstyled = 'true';
+
+  /** @deprecated */
+  dataTestId = null;
+
+  // TODO what/where are these options coming from?!
+  // click: mapEventToAction('onClick', {
+  //   preventDefault: false,
+  //   stopPropagation: true,
```

Comment:
@andrewpye when you have a sec wondering if you can recall where are these event handling options coming from?

Reply:
@vladucu apparently came from fixing a [test](https://github.com/smile-io/ember-polaris/commit/ec399e4f91380cefa4ad78ce40b7594157bd851b) 🤷‍♂

Reply:
thx